lib/adapters/AbstractAdapter.js

```diff
@@ -73,6 +83,16 @@ class AbstractAdapter extends AbstractReaderWriter {
 		});
 	}
 
+	/**
+	 * Validate if virtual path should be excluded
+	 *
+	 * @param {string} virPath Virtual Path
+	 * @returns {boolean} True if path is excluded, otherwise false
+	 */
+	isExcludedPath(virPath) {
```

Comment:
`isPathExcluded` would sound better to me. What do you think?

Reply:
Done